config/workflows/speechToTextWF.xml

```diff
@@ -15,8 +15,12 @@
     <prereq>stt-create</prereq>
     <label>Move new STT files to workspace</label>
   </process>
-  <process name="update-cocina">
+  <process name="process-files">
```

Comment:
This isn't blocking but "process files" sounds pretty generic--maybe a more descriptive name would be helpful? Something like `apply-stoplist`.

Reply:
meant to hold off for more discussion, but i just merged the other PR which named the actual robot ... so I'm going to go with this for now, we can rename later if the team thinks this name is too generic